docs/src/main/tut/README.md

```diff
@@ -152,7 +155,7 @@ Likewise, you can define [gRPC] services in your proto files, with RPC method pa
 // The greeter service definition.
 service Greeter {
   // Sends a greeting
-  rpc SayHello (HelloRequest) returns (HelloReply) {}
+  rpc sayHello (HelloRequest) returns (HelloReply) {}
```

Comment:
This is what the current `protoGen` actually outputs - a fix will address https://github.com/frees-io/sbt-freestyle-protogen/issues/21

Reply:
👍 Fixing frees-io/sbt-freestyle-protogen#21 in the incoming PRs would be great.